packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -484,6 +486,14 @@ export default class TabAdvanced extends Component {
                   icon='vm-force-shutdown'
                   labelId='forceShutdownVmLabel'
                 />
+                <TabButton
+                  btnStyle='warning'
+                  disabled={getXoaPlan().value < PREMIUM.value}
+                  handler={vmWarmMigration}
+                  handlerParam={vm}
+                  icon='vm-warm-migration'
+                  labelId='vmWarmMigration'
+                />
```

Comment:
I think we should also add a small tooltip when the button is disabled to say "Available in XOA Premium" otherwise there's no way to know why it's disabled.